.eslintrc.json

```
@@ -0,0 +1,22 @@
+{
+  "parserOptions": {
+    "project": "./tsconfig.json",
+    "tsconfigRootDir": "./"
+  },
+  "extends": ["plugin:@typescript-eslint/recommended", "standard"],
+  "plugins": ["@typescript-eslint"],
+  "env": {
+    "es6": true
```

Beign this a Node package is good have this too.
```suggestion
    "es6": true,
     "node": true
```